extensions-contrib/aliyun-oss-extensions/pom.xml

```diff
@@ -100,11 +100,6 @@
             <artifactId>jsr305</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-            <scope>provided</scope>
```

Comment:
drive by comment: i'm pretty sure it is important that these are marked as scope provided so the jars don't get duplicated by the extensions when building a distribution. That said, i could also be misremembering, please make sure that there are not multiple copies after this change

Reply:
I build the package with -Pdist. I checked for commons-lang jar in the distribution
I can only find 2 instances

➜  apache-druid-32.0.0-SNAPSHOT git:(commons-lang3) find . -name "commons-lang-*.jar"
./extensions/druid-ranger-security/commons-lang-2.6.jar
./lib/commons-lang-2.4.jar

Reply:
there are no places where only the `scope` gets removed:
* all `pom.xml` changes are either remove the dep
* there is one place where the dependency is upgraded from lang2 to lang3; but there are no scope changes there either